e2e/integration_tests/data_flow_test.go

```diff
@@ -205,7 +205,7 @@ func Test_EndpointLinksAreAvailable(t *testing.T) {
 	}
 
 	ctx := context.Background()
-	endpointlinker.LinkAllOrgsAndEndpoints(ctx, store, false)
+	endpointlinker.LinkAllOrgsAndEndpoints(ctx, store, "", "", false)
```

Comment:
I did this because there is nothing in them, and since the whitelist/blacklist files could contain anything and are subject to change, I figured for the data_flow test it would be better not to use them, especially since I have testing for it elsewhere. If my intuition was wrong I can fix this, just let me know!

Reply:
Added some testing by creating a fake whitelist and blacklist file in the test data for the dataflow test

Reply:
oh cool, the test data was a good idea!